observer/observer.go

```diff
@@ -0,0 +1,73 @@
+package observer
+
+import (
+	blog "github.com/letsencrypt/boulder/log"
+	// _ are probes imported to trigger init func
+	_ "github.com/letsencrypt/boulder/observer/probes/dns"
+	_ "github.com/letsencrypt/boulder/observer/probes/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	statTotalMonitors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_monitors",
+			Help: "count of configured monitors",
+		},
+		[]string{"probe", "name"},
+	)
+	statTotalErrors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_errors",
+			Help: "count of errors encountered by all monitors",
+		},
+		[]string{"probe", "name"},
+	)
+	statTotalObservations = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "obs_observations",
+			Help: "count of observations performed by all monitors",
+		},
+		[]string{"probe", "name"},
+	)
```

Comment:
These can be a single metric with a "success" field which gets set to true or false. Makes analysis later on easier.